apollo-ios/Sources/ApolloAPI/ObjectData.swift

```diff
@@ -11,6 +11,9 @@ public struct ObjectData {
   public let _transformer: any _ObjectData_Transformer
   public let _rawData: [String: AnyHashable]
 
+  public static let _boolTrue = AnyHashable(true)
+  public static let _boolFalse = AnyHashable(false)
```

Comment:
I believe the need for these to be accessible is due to the subscript being `@inlinable`. We should be able to make these `@usableFromInline private` instead of `public`.

Reply:
TIL - that's great to know! They're required to be `@usableFromInline internal` rather than `private` but that achieves what I wanted. Thanks you!